content/docs/start/data-and-model-versioning.md

```diff
@@ -163,11 +163,24 @@ run it after `git clone` and `git pull`.
 If you've run `dvc push`, you can delete the cache (`.dvc/cache`) and
 `data/data.xml` to experiment with `dvc pull`:
 
+<toggle>
+<tab title="Mac/Linux">
+
 ```dvc
 $ rm -rf .dvc/cache
 $ rm -f data/data.xml
 ```
 
+</tab>
+<tab title="Windows">
+
+```
+> del data\data.xml
```

Comment:
Separate question: [Prism ](https://prismjs.com/#supported-languages)doesn't seem to have a Windows Command Prompt language unfortunately. It does have PowerShell but I assume we're going with plain `cmd` right @shcheklein ? If so we may need to make a new custom highlighter cc @julieg18 WDYT

Reply:
@jorgeorpinel I don't think we need a custom highlighter since I think PowerShell runs Windows Command Prompt language as well as its own special syntax. Using the `powershell` language should suffice. 

How this script would look in `powershell` using [Prism's Test drive tool](https://prismjs.com/test.html#language=powershell):

![image](https://user-images.githubusercontent.com/43496356/153596480-80a78259-af97-48c5-b632-6e24a0f53478.png)

Reply:
Good call! But in any case the reason we have the `dvc` highlighter is to highlight `dvc {something}` commands... So we may need to eventually move that to `dvc-posix` and make a new `dvc-win` one, perhaps based of powershell?

We can continue to discuss in https://github.com/iterative/dvc.org/pull/3273